### PR TITLE
1. feat. Nye tabeller for arena migrering adapter

### PR DESCRIPTION
**Pull request by eirikv**

### CODE REVIEW

File: src/main/resources/db/migration/V101__arena_migrering_tabeller.sql [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    created        timestamp    not null,
+    status         arena_status not null,
+    retry_count    int          not null,
+    operation      varchar      not null,
+    operation_time timestamp    not null,
+    payload        jsonb        not null,
+    UNIQUE(arena_id, arena_table)
+);
+
+CREATE INDEX idx_arena_id_table ON arena_event (arena_id, arena_table);
```
Comment (Oddsor) [inline src/main/resources/db/migration/V101__arena_migrering_tabeller.sql:17]: vil ikke UNIQUE lage en index automagisk?

Comment (Oddsor) [reply src/main/resources/db/migration/V101__arena_migrering_tabeller.sql:17]: https://www.postgresql.org/docs/current/sql-createtable.html#SQL-CREATETABLE-PARMS-UNIQUE

> Adding a unique constraint will automatically create a unique btree index on the column or group of columns used in the constraint.

**Review by Oddsor — APPROVED** [review]
🚀

File: src/main/resources/db/migration/V101__arena_migrering_tabeller.sql (hunk 1 above)
Comment (eirikv) [reply src/main/resources/db/migration/V101__arena_migrering_tabeller.sql:17]: Will fix!

**Pull request merged**